### ISSUE TITLE
Why timestamp instead of timestamps?

### ISSUE THREAD

**Issue by JeffreySarnoff** [JuliaStats/TimeSeries.jl#315]
It seems that timestamp(ts) and value(ts) or timestamps(ts) and values(ts) would be more consistent.
(a minor issue)

**Comment by JeffreySarnoff** [comment]
eh

**Comment by milktrader** [comment]
Yeah, not sure how much of others' code it would break. Of course we could go the deprecation route to mitigate that issue.

**Comment by JeffreySarnoff** [comment]
@milktrader thanks for the note .. let's deprecate it -- the advantage is to new users, and naturalness in soft-terms is its own reward.

also ..

**Comment by milktrader** [comment]
We have a `deprecated.jl` file in the `/src` directory. 

Both plural or both singular?

**Comment by milktrader** [comment]
It's more of a stretch to say values are a value than it it to say a timestamp is timestamps so I would vote for going plural.

`timestamp` -> `timestamps`

**Comment by milktrader** [comment]
Also fwiw, the original names come from the name of the type parameters.

`ts.values`
`ts.timestamp`

**Comment by JeffreySarnoff** [comment]
both plural because it is a rare event to have a TimeArray with a single timestamp and a values array that is a vector with one entry

**Comment by milktrader** [comment]
After thinking about this for a few days I'm not sure we need to change the `timestamp` method name to `timestamps`. Here is my reasoning.

`timestamp` is a field of the TimeArray type. The `timestamp` method is nothing more than a wrapper for that field.

````julia
###### element wrapers ###########

timestamp{T,N}(ta::TimeArray{T,N}) = ta.timestamp
values{T,N}(ta::TimeArray{T,N})    = ta.values
colnames{T,N}(ta::TimeArray{T,N})  = ta.colnames
meta{T,N}(ta::TimeArray{T,N})      = ta.meta
````
If anything, I would be in favor of removing this code altogether.

**Comment by JeffreySarnoff** [comment]
There has been an effort to encourage exactly that use of fieldnames as
functional accessors to the verysame field -- and to discourage
immediate use of the .fieldname form outside of very low level ops.  AFAIK
that is still the preference -- but I have not discussed it recently.  Some
of the desire is to protect the specifics of a type implementation from
becoming necessary furnature.
